asv_bench/benchmarks/index_object.py

```diff
@@ -111,6 +112,18 @@ def time_get_loc_dec(self):
         self.idx_dec.get_loc(100000)
 
 
+class IndexEquals:
+    def setup(self):
+        idx_large_fast = RangeIndex(1000000)
```

Comment:
can you make this smaller, we want the original benchmark to not take an excessive amount of time. say 1s at most (IIRC you had it at 10s).